app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -116,7 +124,22 @@ class GithubWebhookSubscription extends SubscriptionHandler {
           await scheduler.cancelPreSubmitTargets(pullRequest: pr, reason: 'Pull request closed');
         } else {
           // Merged pull requests can be added to CI.
-          await scheduler.addPullRequest(pr);
+          log.fine('Pull request ${pr.number} was closed and merged.');
+          final RepositorySlug slug = pr.base!.repo!.slug();
+          final String sha = pr.base!.sha!;
+          final GerritCommit? gobCommit = await gerritService.findMirroredCommit(slug, sha);
+          if (gobCommit != null) {
+            log.fine('Merged commit was found on GoB mirror. Scheduling postsubmit tasks...');
+            await scheduler.addPullRequest(pr);
+          } else {
+            // As of Jan 23, 2023, the retention policy for Pub/Sub messages is
```

Comment:
This seems like important info that should be added to the method's dartdoc instead of inlined here

See also https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis